assets/js/vue/components/search/ResultSet.vue

```diff
@@ -42,13 +42,26 @@
                 </div>
             </div>
         </section>
-        <b-pagination
-                v-model="currentPage"
-                :total-rows="rows"
-                :per-page="formValues.perPage"
-                class="bg justify-content-center"
-                style="margin-bottom: 100px;">
-        </b-pagination>
+
+        <div class="container">
```

Comment:
Maybe this should be a component?

Reply:
I don't see the benefit of making this pagination to be a component, its more work to get events propagated from the child back to the parent, then submit the form in the grandparent component